server/routes/Products.js

```diff
@@ -0,0 +1,43 @@
+//* All routes related to Product's
+
+const express = require("express");
+const Products = require("../models/productSchema");
+const router = express.Router();
+
```

Comment:
Also it would be awesome if you could add a little info here, so that it's easier for other contributors. 
Take a look at this, a format like this can be used !

```js
/**
 * @description Login User
 * @route POST /user/login
 * @access Public
 * @requires email,password
 * @returns token (string) and isuser (boolean)
 */
```